src/midi/midi_alsa.cpp

```diff
@@ -168,30 +180,94 @@ void MidiHandler_alsa::PlayMsg(const uint8_t *msg)
 
 void MidiHandler_alsa::Close()
 {
+	seq_client = 0;
+	seq_port = 0;
```

Comment:
Can we replace these two members with an `alsa_address seq` object, then use:
- `seq.client`
- `seq.port`